assets/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-openstack-cinder-csi-driver
```

Comment:
The NS should be `openshift-cluster-csi-drivers` and it's created by CVO, so the CSI driver operator doesn't need to create it. You can add it to `manifests/` for future use, like the  AWS EBS CSI driver operator does: https://github.com/openshift/aws-ebs-csi-driver-operator/tree/master/manifests

Reply:
done